Utils/hdinsight-node-common/resources/htmlResources/hdinsight/job/html/common.js

```diff
@@ -78,4 +97,10 @@ function formatServerTime(gmtTime) {
     var formatTime = d3.timeFormat("%b %d, %Y %H:%M:%S");
     var ptime = strictIsoParse(gmtTime);
     return formatTime(ptime);
-}
\ No newline at end of file
+}
+
+
+function getTimeIntervalByMins(time1, time2) {
+    var strictIsoParse = d3.utcParse("%Y-%m-%dT%H:%M:%S.%LGMT");
```

Comment:
I will try the module on our new browser. But I think we can just keep it for the following release. What do you think?